src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectRevisionController.java

```diff
@@ -175,4 +176,22 @@ public CatalogObjectMetadata create(
         return catalogObjectMetadataList;
     }
 
+    @ApiOperation(value = "Restore a catalog object revision")
+    @ApiResponses(value = { @ApiResponse(code = 404, message = "Bucket, object or revision not found"),
+                            @ApiResponse(code = 401, message = "User not authenticated"),
+                            @ApiResponse(code = 403, message = "Permission denied") })
+    @RequestMapping(value = "/{commitTimeRaw}", method = PUT)
+    public ResponseEntity<CatalogObjectMetadata> restore(
+            @ApiParam(value = "sessionID", required = false) @RequestHeader(value = "sessionID", required = false) String sessionId,
+            @PathVariable String bucketName, @PathVariable String name, @PathVariable Long commitTimeRaw)
+            throws UnsupportedEncodingException, NotAuthenticatedException, AccessDeniedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)